src/modules/keyboard/Config.cpp

```diff
@@ -502,14 +502,14 @@ QString
 Config::prettyStatus() const
 {
     QString status;
-    status += tr( "Set keyboard model to %1.<br/>" )
+    status += tr( "Keyboard model has been set to %1<br/>", "@label, %1 is keyboard model, as in Apple Magic Keyboard" )
```

Comment:
Is this progress or just informative label?

Same question for the label below.

Reply:
I *did* say "`prettyStatus()` is for what a job is doing (e.g. progress)" but some of them do just one thing, and it's quick so the progress message is just "doing the thing". So this is progress, but "has been set" is fine as well because by the time the user sees it at all, it has been done -- but is that consistent with other `prettyStatus()` messages?

Reply:
I think it's okay to let the user know that it's done, with a full stop. If anything, it makes the progress step more accurate.